test/e2e/storage/persistent_volumes-local.go

```diff
@@ -108,6 +109,19 @@ var setupLocalVolumeMap = map[localVolumeType]utils.LocalVolumeType{
 	BlockFsWithoutFormatLocalVolumeType:     utils.LocalVolumeBlock, // block device in Filesystem mode (default in this test suite)
 }
 
+// setupLocalVolumeMapKeys returns all keys in a sorted slice. This is needed
+// to define tests in a deterministic order.
+func setupLocalVolumeMapKeys() []localVolumeType {
```

Comment:
I was tempted to write some generic helper for this, but then couldn't decide where to put it...